components/select/multiselect.js

```diff
@@ -0,0 +1,94 @@
+import Select from 'react-select'
+
+const selectOptions = [
+  { value: 'angellist', label: 'AngelList' },
+  { value: 'blog', label: 'Blog' },
+  { value: 'event', label: 'Event' },
+  { value: 'linkedin', label: 'LinkedIn' },
+  { value: 'linkedin_connection', label: 'LinkedIn Connection' },
+  { value: 'twitter', label: 'Twitter' },
+  { value: 'wordofmouth', label: 'Word of Mouth' },
+  { value: 'youtube', label: 'YouTube' },
+]
+
+const selectStyles = {
```

Comment:
There are quite a few colors and variables here that are similar to the ones used in the other form elements. We could use a common style object to define them and import them where needed:
- Select imports them and uses them in its object style notation (uses Emotion)
- our internal styles use Styled Components so we can import them and interpolate them in the styled notation

Reply:
Hey Ale, good recommendation, I think its good idea and I can fix it but maybe in another issue/commit since I will start changing other components as well